security/docker/Dockerfile.node-agent-k8s

```diff
@@ -3,11 +3,12 @@ ARG BASE_DISTRIBUTION=default
 
 # The following section is used as base image if BASE_DISTRIBUTION=default
 FROM istionightly/base_debug as default
+RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates && update-ca-certificates && apt-get clean  && rm -rf /var/lib/apt/lists/*
```

Comment:
not sure distroless contains `update-ca-certifcates` package whatever this package is.  @kramerul  can optimize for distroless in master independently of 1.2  requirements.  The next step for distroless IMO is getting one e2e basic test to run against the distroless image per commit.

Cheers
-steve